src/data/wallets/wallet-data.ts

```diff
@@ -0,0 +1,1624 @@
+interface WalletData {
+  last_updated: string
+  name: string
+  image_name: string
+  brand_color: string
+  url: string
+  wallet_live_date: string
+  active_development_team: boolean
+  languages_supported: string[]
+  twitter: string
+  discord: string
+  reddit: string
+  telegram: string
+  ios: boolean
+  android: boolean
+  linux: boolean
+  windows: boolean
+  macOS: boolean
+  firefox: boolean
+  chromium: boolean
+  hardware: boolean
+  open_source: boolean
+  repo_url: string
+  non_custodial: boolean
+  security_audit: string[]
+  scam_protection: boolean
+  hardware_support: boolean
+  walletconnect: boolean
+  rpc_importing: boolean
+  nft_support: boolean
+  connect_to_dapps: boolean
+  staking: boolean
+  swaps: boolean
+  multichain?: boolean
+  layer_2: boolean
+  gas_fee_customization: boolean
+  ens_support: boolean
+  erc_20_support: boolean
+  eip_1559_support: boolean
+  buy_crypto: boolean
+  withdraw_crypto: boolean
+  multisig: boolean
+  social_recovery: boolean
+  onboard_documentation: string
+  documentation: string
+  mpc?: boolean
+}
+
+const walletData: WalletData[] = [
+  {
+    last_updated: "June 22, 2022",
+    name: "Keystone",
+    image_name: "keystone",
+    brand_color: "#ffffff",
+    url: "https://keyst.one/",
+    wallet_live_date: "Dec 2018",
+    active_development_team: true,
+    languages_supported: ["en", "zh", "es", "ko"],
+    twitter: "https://twitter.com/KeystoneWallet",
+    discord: "https://keyst.one/discord",
+    reddit: "",
+    telegram: "https://t.me/KeystoneWallet",
+    ios: true,
+    android: true,
+    linux: false,
+    windows: false,
+    macOS: false,
+    firefox: false,
+    chromium: false,
+    hardware: true,
+    open_source: true,
+    repo_url: "https://github.com/KeystoneHQ",
```

Comment:
This seems to be the only field we append `url` - should we make this consistent (i.e. add to other URL fields or remove from here)?

Reply:
I'd be in favour of being explicit